style.css

```diff
@@ -0,0 +1,616 @@
+}
+
+.about-par {
+  text-align: center;
+  width: 59%;
+  margin: 0 auto;
+  padding-top: 30px;
+}
+
+.about-logo {
+  height: 300px;
+  border: 1px solid #d3d3d3;
+  margin: 20px auto;
+  padding: 80px 150px;
+}
+
+.past-projects {
+  margin: 0 20px;
+  margin-top: 100px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.section-3 {
+  margin-bottom: 40px;
+}
+
+.project-1 {
+  background-image: url("img/proj-1.png");
+  width: 30%;
+  height: 150px;
+  margin: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.project-2 {
+  background-image: url("img/proj-2.jpg");
+  width: 30%;
+  height: 150px;
+  margin: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.bold {
+  color: #272a31;
+  font-weight: bolder;
+  font-size: 35px;
+}
+
+.bottom-text {
+  color: #d3d3d3;
+}
+
+.main-section {
+  background-image: url("img/grey\ img.png");
+  text-align: center;
+  padding-top: 20px;
+  background-size: cover;
+  padding-bottom: 50px;
+}
+
+.main-section-h1 {
+  text-align: center;
+  margin-bottom: 10px;
+  color: white;
+}
+
+.orange-underline {
+  width: 70px;
+  margin: 0 auto;
+  border-bottom: 5px solid #ec5242;
+  margin-bottom: 20px;
+}
+
+.blocks {
+  display: flex;
+  list-style: none;
+  margin: 10px 10%;
+  margin-bottom: 50px;
+}
+
+.blocks li {
+  margin: 10px;
+  background-color: #3d3a3a7c;
+  padding: 10px 0;
+}
+
+.blocks h2 {
+  color: #ec5242;
+}
+
+.blocks p {
+  color: #d3d3d3;
+}
+
+.see-more {
+  color: #d3d3d3;
+}
+
+.fas {
+  color: #d3d3d3;
+  padding: 10px 10px;
+}
+
+.speaker-header {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.speaker-list {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-top: 60px;
+  padding: 10px 10%;
+}
+
+.speaker {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  width: 45%;
+  margin: 15px;
+}
+
+.speaker img {
+  height: auto;
+  width: 150px;
+}
+
+.grey-underline {
+  width: 30px;
+  margin-top: 15px;
+  border-bottom: 2px solid grey;
+}
+
+.job-position {
+  color: #ec5242;
+  font-style: italic;
+}
+
+.partner {
+  text-align: center;
+  background-color: #534f4e;
+  padding: 40px 10px;
+}
+
+.partner-header {
+  margin-bottom: 5px;
+  color: white;
+}
+
+.bottom-logos {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  padding: 10px 20px;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.bottom-logos img {
+  height: 50px;
+  filter: grayscale(100%) contrast(30%);
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+  padding: 40px 10%;
+  align-items: center;
+}
+
+.bg-color {
+  background-color: #272a31;
+}
+
+.white-text {
+  color: white;
+}
+
+.footer img {
+  height: 190px;
+}
+
+.footer-about {
+  margin-top: 10px;
+}
+
+.footer-text {
+  display: flex;
+  flex-direction: column;
+  color: #d3d3d3;
+  font-size: 11px;
+}
+
+@media (max-width: 767px) {
```

Comment:
* I suggest you apply mobile-first design philosophy  as it creates a better experience for users and its easy to scale up the design to suit any device screen size [Read more](https://anglestudios.co.uk/blog/why-mobile-first-web-design-is-becoming-more-important/#:~:text=Mobile%2Dfirst%20ensures%20that%20you,strong%20base%20to%20work%20from.)

Reply:
Thank you for the feedback I have made the necessary changes